src/js/cbLink.jsx

```diff
@@ -0,0 +1,12 @@
+var React = require('react');
```

Comment:
The main purpose here was as a place for routing to hook into, or for common onClick code to be executed (like e.preventDefault()). Not strictly needed, but I think as we implement routing we'll want something like this.